stdlib/src/builtin/string.mojo

```diff
@@ -1439,6 +1439,51 @@ struct String(
 
         return output
 
+    fn splitlines[keepends: Bool = False](self) -> List[String]:
```

Comment:
Please follow the Python API for this. We can optimize later.
```suggestion
    fn splitlines(self, keepends: Bool = False) -> List[String]:
```

Reply:
Done!